### PR TITLE
docs: Added DLT integration

### PR DESCRIPTION
**Pull request by Anush008**
Adds Qdrant-DLT integration to the integration section of the docs.

New Section Preview:
https://deploy-preview-382--condescending-goldwasser-91acf0.netlify.app/documentation/integrations/dlt/

### CODE REVIEW

**Comment by Anush008** [comment]
Waiting on https://github.com/dlt-hub/dlt/pull/724 to get merged.

**Comment by Anush008** [comment]
@kacperlukawski, review 🙏.

**Comment by Anush008** [comment]
DLT doc about the integration.
https://dlthub.com/docs/dlt-ecosystem/destinations/qdrant

File: qdrant-landing/content/documentation/integrations/dlt.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+A DLT [write disposition](https://deploy-preview-724--dlt-hub-docs.netlify.app/docs/dlt-ecosystem/destinations/qdrant/#write-disposition) defines how the data should be written to the destination. All write dispositions are supported by the Qdrant destination.
+
+## DLT Sync
+
+Qdrant destination supports syncing of the [`DLT` state](https://dlthub.com/docs/general-usage/state#syncing-state-with-destination).
+
+## Next steps
+
+- The comprehensive Qdrant DLT destination documentation can be found [here](https://deploy-preview-724--dlt-hub-docs.netlify.app/docs/dlt-ecosystem/destinations/qdrant/).
```
Comment (kacperlukawski) [inline qdrant-landing/content/documentation/integrations/dlt.md:101]: The link points to the preview. Can we change it to the main website?

File: qdrant-landing/content/documentation/integrations/dlt.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+[DLT](https://dlthub.com/) is an open-source library that you can add to your Python scripts to load data from various and often messy data sources into well-structured, live datasets.
+
+With the DLT-Qdrant integration, you can now select Qdrant as a DLT destination to load data into.
+
+**DLT Enables**
+
+- Automated maintenance - with schema inference, evolution, and alerts, and with short declarative code, maintenance becomes simple.
+- Run it where Python runs - on Airflow, serverless functions, notebooks. No external APIs, backends or containers, scales on micro and large infrastructure alike.
+- User-friendly, declarative interface that removes knowledge obstacles for beginners while empowering senior professionals.
```
Comment (kacperlukawski) [inline qdrant-landing/content/documentation/integrations/dlt.md:16]: Is that part extracted from their website? Sounds like some strong marketing :sweat_smile:

**Review by kacperlukawski — CHANGES_REQUESTED** [review]
Could we please remove displaying all the steps as a list?

image

It is not consistent with the other integration descriptions.

File: qdrant-landing/content/documentation/integrations/dlt.md (hunk 2 above)
Comment (Anush008) [reply qdrant-landing/content/documentation/integrations/dlt.md:16]: Yes. From the site and docs. I toned it down along with the other changes.
https://github.com/qdrant/landing_page/pull/382/commits/25b4bd9b210912d63c4599edfa5dfff1616a6274

**Review by kacperlukawski — APPROVED** [review]
LGTM

**Pull request merged**